prismic-model/js/parts/article-body.js

```diff
@@ -290,6 +294,15 @@ export default {
           },
         },
       },
+      tagList: slice('Tag List', {
+        nonRepeat: {
+          title: heading('Title', 2),
+        },
+        repeat: {
+          link: link('Link', 'web'),
+          linkText: text('Link text'),
+        },
+      }),
```

Comment:
Nice, it might be good to take a look at the tag type to update it to something like:
```ts
type Tag {
  link: WebLink;
  linkText: string;
}
```

Looking at it though, these won't support what we have called [`textParts`](https://github.com/wellcomecollection/wellcomecollection.org/blob/31b47efd0d99561d85e0c5538dfe18dbf08ea18b/common/views/components/Tags/Tags.tsx#L8), are we good with that?